corehq/apps/app_manager/static/app_manager/js/detail-screen-config.js

```diff
@@ -494,7 +497,9 @@ var DetailScreenConfig = (function () {
                 hasNodeset: false,
                 nodeset: "",
             };
-            _.defaults(this.original, tabDefaults);
+            _.each(_.keys(tabDefaults), function(key) {
+                that.original[key] = that.original[key] || tabDefaults[key];
+            });
```

Comment:
Underscore 1.5 updated defaults to only override undefined values. Previously it had been all falsy values, which is what this code needs (it's largely overwriting nulls, not undefined).